src/plugins/audioPlugin.js

```diff
@@ -13,7 +13,7 @@ export default function createAudioPlugin() {
 
   const embedToHTML = embed => {
     const { audio: { title, audioFile: { mimeType, url } } } = embed;
-    const replacement = `<figure class="article_audio"><audio controls type="${mimeType}" src="${url}"></audio><figcaption>${title}</figcaption></figure>`;
+    const replacement = `<figure class="article_audio"><audio controls style="width: 100%" type="${mimeType}" src="${url}"></audio><figcaption>${title.title}</figcaption></figure>`;
```

Comment:
Er dette en måte å omgå reglene med at vi ikke kan gjøre ting i frontend-packages lengre? :p Skulle det egentlig vært en egen css-klasse for dette, eller er det vel så bra å bruke inline styling akkurat her?

Reply:
Nja, vi har ikke noe design på hvordan lyd elementer skal se ut 😛 . Det burde vært stylet med egen css klasse, men er bare en quick fiks jeg la til under testing for å se om det ble seende bedre ut. Glemte å ta den bort igjen. Fjerner den før jeg merger.